docs/releases/breaking-changes.md

```diff
@@ -15,13 +15,15 @@ Starting with Directus 10.0, here is a list of potential breaking changes with r
 
 ## Version 10.8.3
 
-### Updated GraphQL content version usage
+### Updated GraphQL Content Version Usage
 
-Content versioning was originally implemented as a GraphQL parameter however the output format for relational fields is
-different for stored versions resulting in a GraphQL error as this is unsupported.
+Previously when accessing content versions via GraphQL, a `version` parameter on existing fields. This has now been
```

Comment:
This sentence seems incomplete

Reply:
It's finished on the next line, as per the linter

Reply:
> Previously when accessing content versions via GraphQL, a `version` parameter on existing fields.

Hmm, I'm missing a predicate here, for example:

> Previously when accessing content versions via GraphQL, a `version` parameter on existing fields **was used**.

Reply:
Oh. Um. Nothing to see here, just my lack of ability to read.

Reply:
Admittedly, GitHub diffs are not exactly the best tool for reviewing prose text 😅